### ISSUE TITLE
Fresh install, megadesk only beeps whatever I do. 

### ISSUE THREAD

**Issue by Extenuate0083** [gcormier/megadesk#147]
I just installed my new megadesk but it keeps beeping. Both buttons make different sounds so I'm pretty sure I've hooked everything up correctly but I can't press 16 times on the up button because it only beeps. Nothing else happens.

Please help!

![PXL_20240109_232715282 MP](https://github.com/gcormier/megadesk/assets/132627587/af1c234a-aec0-4ab6-adb9-0d9e1cf760cc)

https://github.com/gcormier/megadesk/assets/132627587/619b17fa-4b2b-4dbb-8531-5060860f8458

**Comment by gcormier** [comment]
16x is no longer needed in the newer boards.

Have you tried removing unplugging your desk from the wall until it's dead and plugging it back in?

**Comment by Extenuate0083** [comment]
Yes! Several times.

**Comment by gcormier** [comment]
Maybe it got damaged in transit - can you reach out to me on Tindie with your order #?

**Comment by Extenuate0083** [comment]
Absolutely! Is there something else I could try to see if the problem is on
my side?
